### PR TITLE
Write graphite guide in realtime section

### PR DESCRIPTION
**Pull request by shblue21**
## Description

Write graphite setting guide docuementation in realtime section, with simple pickle protocol

## Motivation and Context

Improved accessibility for Jmeter and Graphite integration

## How Has This Been Tested?

After building with gradle site, check the screen by resolution.

## Screenshots (if appropriate):
image

## Types of changes

- Write docs

## Checklist:

- [x] My code follows the [code style][style-guide] of this project.
- [x] I have updated the documentation accordingly.

[style-guide]: https://wiki.apache.org/jmeter/CodeStyleGuidelines

### CODE REVIEW

File: xdocs/usermanual/realtime-results.xml [hunk 1]
```diff
@@ -186,10 +186,22 @@ InfluxDB data can be easily viewed in a browser through <a href="http://grafana.
     <figure width="1265" height="581" image="grafana_dashboard.png">Grafana dashboard</figure>
 </subsection>
 
-<subsection name="&sect-num;.5 Graphite" anchor="graphite">
-<p>HELP WELCOME for this section, see <a href="../building.html" >Contributing documentation</a></p>
+<subsection name="&sect-num;.5 Graphite Configuration" anchor="graphite_configuration">
+    <p>To make JMeter send metrics to backend add a BackendListener using the GraphiteBackendListenerClient.</p>
+    <p><a href="./component_reference.html#Backend_Listener">GraphiteBackendListenerClient</a> section will help you confiugration.</p>
```
Comment (pmouawad) [inline xdocs/usermanual/realtime-results.xml:191]: will help you do the configuration

File: xdocs/usermanual/realtime-results.xml [hunk 2]
```diff
@@ -186,10 +186,22 @@ InfluxDB data can be easily viewed in a browser through <a href="http://grafana.
...
+    <figure width="902" height="341" image="backend_listener_graphite.png">Graphite configuration</figure>
 </subsection>
 
+<subsection name="&sect-num;.5.1 Graphite Sender" anchor="graphite_sendor">
```
Comment (pmouawad) [inline xdocs/usermanual/realtime-results.xml:195]: Remove "Can use"

**Review by pmouawad — CHANGES_REQUESTED** [review]
Thank you for your PR.
Can you fix the typos so that I can merge it ?
Regards

**Comment by shblue21** [comment]
Thanks for the feedback. I am happy to be able to contribute in this way. jmeter is a great tool.

**Review by pmouawad — APPROVED** [review]

**Comment by FSchumacher** [comment]
Thanks for the PR. I rephrased the wording a bit and removed space at the end of line.

**Pull request closed without merging**